account_financial_report_qweb/report/general_ledger.py

```diff
@@ -890,6 +892,28 @@ def _inject_line_not_centralized_values(
     m.name AS entry,
     j.code AS journal,
     a.code AS account,
+    CASE
+        WHEN
+            ml.tax_line_id is not null
+        THEN
+            at.name
```

Comment:
use `at.description` as in most localisation it is shorter than `at.name`. Or `coalesce(at.description, at.name)` which would more or less match what is done on the invoice template.